11-if/questions/5-short-if-edit.md

```diff
@@ -1,84 +1,83 @@
-## How to fix this program?
+# Questions: Short If #
+
+## 1. Which of the choices below would fix the following program? ##
 ```go
 package main
 
 import (
-	"fmt"
-	"time"
+    "fmt"
+    "time"
 )
 
 func main() {
-	if err != nil; d, err := time.ParseDuration("1h10s") {
-		fmt.Println(d)
-	}
+    if err != nil; d, err != time.ParseDuration("1h10s") {
+        fmt.Println(d)
+    }
 }
-``` -
-1. Swap the simple statement with the `err != nil` check. *CORRECT*
-2. Remove the error handling.
-3. Remove the semicolon.
-4. Change the short declaration to an assignment.
+```
+(A) Swapping the simple statement with `err != check`. <br>
+(B) Removing the error handling. <br>
+(C) Removing the remove the semicolon. <br>
+(D) Changing the short declaration to an assignment. <br>
+
+---
 
-> **1:** Yes. In a short if statement, the simple statement (the short declaration there) should be the first part of it. Then, after the semicolon separator, there should be a condition expression.
-> 
-> **2:** You don't want that. That's not the issue here.
+**Solution**: (A) <br>
+When initalizing the short if-statement, the simple statement (the short declaration short come first. Only after the semicolon should the separator should there be a conditional expression.
```

Comment:
Fixed:
```**Solution**: (A)

That's right! When initializing short if-statements, the simple statement (the short-declaration) should come first. Only after the semicolon should there be a conditional expression.

Removing the error handler is not the issue and semicolons are valid end-of-statement separators in Go. (D) Changing the short declaration to an assignment declaration would not work because then it would no longer be conditional.
```